sdk/extensions/Microsoft.Extensions.Azure/CHANGELOG.md

```diff
@@ -4,6 +4,16 @@
 
 ### Features Added
 
+- Added support for [federated managed identity](https://learn.microsoft.com/entra/workload-id/workload-identity-federation-config-app-trust-managed-identity?tabs=microsoft-entra-admin-center#azureidentity) support in the client factory by specifying configuration item `credential` as "managedidentityasfederatedidentity" and providing the following named configuration items:
+
+  - `tenantId` : The tenant where the target resource was created
+  - `clientId` : The client identifier for the application, which must be granted access on the target resource
+  - `managedIdentityClientId` : The managed identity which you configured as a Federated Identity Credential (FIC)
+  - `federatedAudience`: One of the following token audiences, specific to the cloud that you're running in:
```

Comment:
Should we consider making this value something like `AudienceCloud` and accept values like `global` (default), `usgov`, `china` so that users don't need to remember the magic audience strings?

Reply:
I'm good with that, but its @scottaddie's call.   I followed the pattern that he asked for and I'm not sure if there are any wider considerations.

Reply:
Changed to `azureCloud`